packages/data-service-generator/src/server/custom-module/create-custom-module.ts

```diff
@@ -0,0 +1,119 @@
+import { plural } from "pluralize";
+import { camelCase } from "camel-case";
+import { ModuleContainer, ModuleMap } from "@amplication/code-gen-types";
+import {
+  createServiceId,
+  createServiceModules,
+} from "./custom-service/create-service";
+import { createCustomModuleControllerModules } from "./custom-controller/create-controller";
+import { createCustomModuleResolverModules } from "./custom-resolver/create-resolver";
+import DsgContext from "../../dsg-context";
+import { createCustomModule } from "./custom-module/create-module";
+
+export async function createCustomModulesModules(
+  dtoNameToPath: Record<string, string>
+): Promise<ModuleMap> {
+  const context = DsgContext.getInstance;
+  const { moduleContainers } = context;
+
+  const customModuleContainers = moduleContainers.filter(
+    (module) => module.entityId === undefined
+  );
+
+  const customModules = new ModuleMap(DsgContext.getInstance.logger);
+  for await (const customModule of customModuleContainers) {
+    await customModules.merge(
+      await createCustomModuleModules(customModule, dtoNameToPath)
+    );
+  }
+
+  return customModules;
+}
+
+async function createCustomModuleModules(
```

Comment:
Just a suggestion, not a must, but the naming is very confusing with `createCustomModuleModules` and `createCustomModulesModules`.
Perhaps rename this one to something like `createSingleCustomModuleModules` just for easier readability?